demosplan/DemosPlanCoreBundle/Controller/Survey/SurveyController.php

```diff
@@ -131,7 +129,6 @@ public function createAction(
      *     name="dplan_survey_edit",
      *     methods="GET",
      *     path="/verfahren/{procedureId}/umfrage/{surveyId}/edit")
-     *
```

Comment:
I would prefer to leave an empty comment line between the route and Dplanpermission. Makes the distinction clearer and improves readability. Will become obsolete with Attributes anyway, but for now I would like to keep it as is

Reply:
That's hard to configure, if not impossible.